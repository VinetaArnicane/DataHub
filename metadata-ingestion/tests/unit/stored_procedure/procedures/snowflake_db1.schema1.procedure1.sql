BEGIN
  CREATE TABLE GSL_TEST_DB.PUBLIC.new_table_1 as select * from GSL_TEST_DB.PUBLIC.processed_transactions;
  RETURN message;
END;