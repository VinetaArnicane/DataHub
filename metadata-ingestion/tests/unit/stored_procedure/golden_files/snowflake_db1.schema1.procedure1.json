[
{
    "entityType": "dataJob",
    "entityUrn": "urn:li:dataJob:(urn:li:dataFlow:(snowflake,db1.schema1.stored_procedures,PROD),procedure1)",
    "changeType": "UPSERT",
    "aspectName": "dataJobInputOutput",
    "aspect": {
        "json": {
            "inputDatasets": [
                "urn:li:dataset:(urn:li:dataPlatform:snowflake,gsl_test_db.public.processed_transactions,PROD)"
            ],
            "outputDatasets": [
                "urn:li:dataset:(urn:li:dataPlatform:snowflake,gsl_test_db.public.new_table_1,PROD)"
            ],
            "fineGrainedLineages": []
        }
    }
}
]